src/Hydration.js

```diff
@@ -0,0 +1,39 @@
+class Hydration {
+  constructor(hydrationData) {
+    this.hydrationData = hydrationData;
+  }
+  averageWaterConsumedPerDay(userID) {
+    let userHydrationData = this.hydrationData.filter(el => {
+      return el.userID === userID
+    })
+    return userHydrationData.reduce((acc, el) => {
+      acc += el.numOunces;
+      return acc;
+    }, 0) / userHydrationData.length
+  }
+
+  waterConsumedForSpecificDay(userID, date) {
+    return this.hydrationData.find(el => {
+      return (el.userID === userID) && (el.date === date)
+    }).numOunces
+  }
+
+  waterConsumedForSpecificWeek(userID, endDate) {
+    const date = new Date(endDate)
+    const sevenDaysAgo = new Date(new Date(endDate).setDate(new Date(endDate).getDate() - 7))
```

Comment:
I'm glad you got this part working!
If I'm not mistaken, it looks like this line could be shortened by using that date variable you declared above it.
`const sevenDaysAgo = new Date(date.setDate(date.getDate() - 7))`